documentation/source/install.rst

```diff
@@ -85,7 +85,7 @@ Installation of cocotb
 
     .. parsed-literal::
 
-        pip install cocotb =~ |version|
+        pip install "cocotb~=|version|""
```

Comment:
Is that a mismatched double close quote?

Reply:
Yep. Damn editor.